app/assets/stylesheets/_sprites.scss

```diff
@@ -5,21 +5,21 @@
 /*---------------------- Icons 20 x 20 black ------------------------*/
 
 @font-face {
-    font-family: 'icomoon';
-    src:    font-url('icomoon.eot');
-    src:    font-url('icomoon.eot?#iefix') format('embedded-opentype'),
-        font-url('icomoon.ttf') format('truetype'),
-        font-url('icomoon.woff') format('woff'),
-        font-url('icomoon.svg#icomoon') format('svg');
-    font-weight: normal;
-    font-style: normal;
+  font-family: 'icomoon';
+  src:
+    font-url('icomoon.ttf') format('truetype'),
+    font-url('icomoon.woff') format('woff'),
+    font-url('icomoon.svg#icomoon') format('svg');
```

Comment:
I simplified the font-face declaration and removed support for IE8. We could probably remove the TTF and SVG formats, too from the font-face declaration, too. All browsers (including IE9) support the WOFF format.

We could create a WOFF2 version, which would be half the size of the WOFF version (7KB instead of 16KB).